src/output/dig.rs

```diff
@@ -61,9 +62,14 @@ pub fn write(
                     Expire(expire) => {
                         writeln!(target, "; EXPIRE: {}", expire)?
                     }
-                    TcpKeepalive(opt) => {
-                        writeln!(target, "; TCPKEEPALIVE: {}", opt)?
-                    }
+                    TcpKeepalive(opt) => writeln!(
+                        target,
+                        "; TCPKEEPALIVE: {} secs",
+                        opt.timeout().map_or("".to_string(), |t| format!(
+                            "{:.1}",
+                            Duration::from(t).as_secs_f64()
+                        ))
```

Comment:
Small suggestion here. Dig prints the TCPKEEPALIVE without "secs" if the value is missing. It also has a space between the two words.

```
; <<>> DiG 9.18.33 <<>> google.com A +tcp +keepalive +qr
;; global options: +cmd
;; Sending:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 32748
;; flags: rd ad; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
; COOKIE: 916ce5e901a627c4
; TCP KEEPALIVE:
;; QUESTION SECTION:
;google.com.                    IN      A
...
```